types/react-relay/relay-hooks/useMutation.d.ts

```diff
@@ -5,20 +5,45 @@ import {
     IEnvironment,
     MutationConfig,
     MutationParameters,
+    NormalizationOperation,
     PayloadError,
+    ReaderFragment,
+    RequestParameters,
     SelectorStoreUpdater,
     UploadableMap,
     VariablesOf,
 } from 'relay-runtime';
 
+export interface MutationOperation {
+    fragment: ReaderFragment;
+    kind: string;
+    operation: NormalizationOperation;
+    params: RequestParameters;
+    hash: string;
+}
```

Comment:
Feel this is just;
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/8d09d299ac390d747eb04c4a3951a2f5b295386a/types/relay-runtime/lib/util/RelayConcreteNode.d.ts#L10-L15

Think `hash` is something we need to add into that basetype. Do you mind just using that — and we'll double check the `hash` field.

Reply:
Done. This is causing issues with the tests for `relay-runtime`. The `hash` value would have to be added [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/relay-runtime/relay-runtime-tests.tsx#L306). What value should I set it to?